spec/support/matchers/input_matcher.rb

```diff
@@ -1,10 +1,10 @@
 module Yuriita
   module Matchers
 
-    def an_input_matching(qualifier, term)
+    def an_expression_input_matching(qualifier, term)
```

Comment:
I feel like this should remain `an_input_matcher`. It checks to ensure an expression input or a scope input matches a given input. It doesn't really distinguish between the two.

For example, I would expect this to still be true with this change.

```ruby
scope_input = Input.new(qualifier: "in", term: "title")
query = Query.new(scope_inputs: [scope_input])

expect(query.scope_inputs).to contain_exactly(an_expression_input_matching("in", "title"))
```

In this example, the matcher says "expression_input" but it's checking a "scope input".

Reply:
This is an excellent point! The name is changing back.